File: CPP/suma_cyfr.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  suma_cyfr.py
#  

def main(args):
    
    liczba = int(input("Podaj liczbę: "))
    while liczba < 10:
        print("Złe dane")
        liczba = int(input("Podaj liczbę: "))
    else:
        cyfry = list(map(int, str(liczba)))
    print ("Suma cyfr liczby:", sum(cyfry))

    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: CPP/suma-cyfr.cpp
/*
 * suma-cyfr.cpp
 */


#include <iostream>

using namespace std;

int sumuj_cyfry1(int liczba) {
    
    int suma = 0;
    while(liczba > 0) {
        suma += liczba % 10;
        liczba = liczba / 10;
        }
        return suma;
    }



int main(int argc, char **argv)
{
    
    int liczba = 0;
    while (liczba < 10) {
        cout << "Podaj dowolną liczbę dwucyfrową: ";
        cin >> liczba;
    }
    
    cout << "Suma: " << sumuj_cyfry1(liczba) << endl;
    
    
    return 0;
}

